.github/workflows/main-build.yml

```diff
@@ -0,0 +1,50 @@
+name: Build image and push to ghcr.io
+
+on:
+  workflow_run:
+    workflows: ["Path tests"]
+    types:
+      - completed
```

Comment:
I think this needs to specify a successful run, not just completed.

eg:
```yaml
on:
  workflow_run:
    workflows: ["Path tests"]
    types:
      - completed
    if: ${{ github.event.workflow_run.conclusion == 'success' }}
    ```

Reply:
This has been addressed.